Makefile

```diff
@@ -18,6 +18,7 @@ install:
 .PHONY: test
 test:
 	go test $(TEST_OPTIONS) ./...
+	go clean .
```

Comment:
we already have a clean command in the below, I think we can just use it?

Reply:
I thought as well. I'm a bit lazy to run `make clean` 😅

Reply:
I mean we should use `make clean` here so we don't need to duplicate the logic for cleaning.

Reply:
I understood. I'd close the PR.

Reply:
No.... I mean we can put “make clean” in this line, so we are reusing the logic defined below. Can you make the change and reopen the PR?

Reply:
Too hastened 😅I reopen and update it